src/calculateRentalCost.js

```diff
@@ -28,7 +28,25 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  const dayRentCost = 40;
+  let totalCost = days * dayRentCost;
+
+  const smallDiscount = 20;
+  const bigDiscount = 50;
+
+  if (days >= 7) {
+    totalCost -= bigDiscount;
+
+    return totalCost;
```

Comment:
No need to reassign value, just return calculation result
```suggestion
    
    return totalCost - bigDiscount;
```